checkers/gameLogic.ts

```diff
@@ -948,3 +993,18 @@ export function getInitialBoard(): Board {
     ['WM', '--', 'WM', '--', 'WM', '--', 'WM', '--'],
   ];
 }
+// function isPosOnHintLine(row: number, col: number, hint: RiddleData) {
```

Comment:
I would remove unnecessary comments at the time of a PR

Reply:
it is TODO for the next pull request, we have to finish the work on the riddles, so in the next PR we will fix it.
thanks